server/repository/src/db_diesel/form_schema_row.rs

```diff
@@ -93,15 +95,18 @@ impl<'a> FormSchemaRowRepository<'a> {
     }
 
     #[cfg(not(feature = "postgres"))]
```

Comment:
line 86 also needs updating - currently won't compile when postgres feature is enabled

```
    pub fn upsert_one(&self, schema: &FormSchemaJson) -> Result<(), RepositoryError> {
```